assets/src/edit-story/components/canvas/page.js

```diff
@@ -33,22 +33,29 @@ const Selection = styled.div`
 	position: absolute;
 `;
 
-const Element = styled.div`
-	cursor: pointer;
-	user-select: none;
-`;
-
 function Page() {
 	const [ targetEl, setTargetEl ] = useState( null );
+	const [ clickHandlers, setClickHandlers ] = useState( {} );
+	const moveable = useRef();
+
+	const setClickHandler = useCallback(
+		( id, handler ) =>
+			setClickHandlers( ( handlers ) => ( { ...handlers, [ id ]: handler } ) ),
+		[ setClickHandlers ] );
+
+	const getClickHandler = useCallback(
+		( id ) => clickHandlers[ id ] || ( () => {} ),
+		[ clickHandlers ] );
```

Comment:
This part about click handlers in particular is a bit messy. We might be able to do it in a better way.

And maybe @miina's efforts will make this completely superfluous. It is needed in the version as Moveable is working on a selection object next to the actual objects, but as I understand it, Moveable will in the future work on the actual objects and thus we hopefully don't need this propagation of clicks from one element to another.